web3/trace.py

```diff
@@ -0,0 +1,77 @@
+from web3 import formatters
+from web3.utils.empty import (
+    empty,
+)
+
+class Trace(object):
+    def __init__(self, web3):
+        self.web3 = web3
+
+    defaultAccount = empty
+    defaultBlock = "latest"
+
+    def call(self, transaction, types, block_identifier=None):
+        formatted_transaction = formatters.input_transaction_formatter(self, transaction)
```

Comment:
This function has been refactored away. It looks like adding a two new request formatters for `'trace_call'` to `web3/middleware/pythonic.py` is the right path forward here.